### PR TITLE
workflows updated with newer tools, added missing

### PR DESCRIPTION
**Pull request by astrovsky01**

### CODE REVIEW

**Comment by astrovsky01** [comment]
How do I lint a workflow? I took what came from downloading off of an instance.

**Comment by bgruening** [comment]
```
source /home/travis/miniconda3/bin/activate galaxy_training_material && \
	bash bin/validate-json.sh && \
	bash bin/validate-workflow-tags.sh
```

**Comment by bedroesb** [comment]
@astrovsky01 Since #1724 and #1715 it is mandatory for a workflow on GTN to have the correct tags and annotation (This started during this PR). I will add this workflow linting to planemo hopefully this week. You could fix this travis error easily by editing the workflow file. Make sure following attributes are filled in like this:
```json
"annotation": "Unicycler Assembly",
...
...
"tags": [	    
    "assembly"
  ]
```

**Comment by astrovsky01** [comment]
@bedroesb thank you, I appreciate it!

**Comment by bedroesb** [comment]
@astrovsky01 no problem ! We hope that if these workflows are than added to an instance, people can find them back easier.

**Comment by almahmoud** [comment]
Probably not necessary, but I'd also suggest changing the names to remove the numbering we're using internally (i.e. WF2, WF4) and just match the tutorial name

**Comment by bgruening** [comment]
@astrovsky01 have you checked that those workflows also follow the training material? Do we need to adjust the hands-on material because of parameter changed etc...?

**Comment by astrovsky01** [comment]
@bgruening they don't change the training, except the ones I added to the PR. It's more about updating tools to most recent versions.

File: topics/assembly/tutorials/unicycler-assembly/workflows/unicycler.ga [hunk 1]
```diff
@@ -1,611 +1,590 @@
 {
-    "a_galaxy_workflow": "true",
-    "annotation": "Unicycler Assembly",
-    "format-version": "0.1",
-    "name": "Unicycler training",
+    "a_galaxy_workflow": "true", 
+    "annotation": "Unicycler Assembly", 
+    "format-version": "0.1", 
+    "name": "Unicycler training (imported from uploaded file)",
```
Comment (bgruening) [inline topics/assembly/tutorials/unicycler-assembly/workflows/unicycler.ga]: can you remoe the brackets?

File: topics/introduction/tutorials/galaxy-intro-101-everyone/workflows/main_workflow.ga [hunk 2]
```diff
@@ -1,351 +1,435 @@
...
-      "name": "Input dataset",
-      "label": "iris",
-      "inputs": [
-        {
-          "name": "iris",
-          "description": ""
+    "a_galaxy_workflow": "true", 
+    "annotation": "", 
+    "format-version": "0.1", 
+    "name": "GTN Training: Galaxy 101 For Everyone (imported from uploaded file)",
```
Comment (bgruening) [inline topics/introduction/tutorials/galaxy-intro-101-everyone/workflows/main_workflow.ga]: here as well

Comment (astrovsky01) [reply topics/introduction/tutorials/galaxy-intro-101-everyone/workflows/main_workflow.ga]: +1

File: topics/assembly/tutorials/unicycler-assembly/workflows/unicycler.ga (hunk 1 above)
Comment (astrovsky01) [reply topics/assembly/tutorials/unicycler-assembly/workflows/unicycler.ga]: +1

**Comment by astrovsky01** [comment]
@bgruening  Does anything else need to be  changed?

**Comment by bgruening** [comment]
Sorry for the delay, didn't get any notification for the last 2 weeks, trying to catch up.

**Pull request merged**